lib/Runtime/Library/JavascriptBigInt.cpp

```diff
@@ -71,28 +88,59 @@ namespace Js
         return true;
     }
 
+    bool JavascriptBigInt::Resize(digit_t length, ScriptContext * scriptContext)
+    {
+        digit_t *digits;
+
+        if (length <= m_maxLength)
+        {
+            return true;
+        }
+
+        length += length;// double size
+        if (SIZE_MAX / sizeof(digit_t) < length) // overflow 
+        {
+            return false;
+        }
+
+        digits = RecyclerNewArrayLeaf(scriptContext->GetRecycler(), digit_t, length);
```

Comment:
I think you could use GetScriptContext()->GetRecycler() to avoid passing in the ScriptContext. Ideally the only case we would need to pass the script context is in CloneToScriptContext, which is explicitly requesting a new copy of the object in some other script context (not the script context it was created from).

Reply:
will do